setup.py

```diff
@@ -48,7 +48,7 @@ def find_version(*file_paths):
     include_package_data=True,
     # See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
     entry_points={"sphinx.html_themes": ["pydata_sphinx_theme = pydata_sphinx_theme"]},
-    install_requires=["sphinx", "beautifulsoup4", "docutils<0.17"],
+    install_requires=["sphinx", "beautifulsoup4", "docutils<0.18"],
```

Comment:
wouldn't something like this be safer?

```suggestion
    install_requires=["sphinx", "beautifulsoup4", "docutils!=0.17.0"],
```

Reply:
I didn't even know that syntax was possible lol

Reply:
TBH now I'm unsure, I know it works for pip requirements but maybe not for setup.py?